src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
Thanks for linking your code 👍 Actually I quickly jumped by the TA desk to ask for it but you were already gone. We'll see if we can include it but its beyond the scope of this PR and we will evaluate when we will make this enhancement!